src/SFA.DAS.EAS.Account.Worker/Infrastructure/AzureContainerRepository.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Queue;
+using SFA.DAS.EAS.Account.Worker.Infrastructure.Interfaces;
+using SFA.DAS.EAS.Domain.Interfaces;
+
+namespace SFA.DAS.EAS.Account.Worker.Infrastructure
+{
+	public class AzureContainerRepository : IAzureContainerRepository
```

Comment:
Could this be renamed to reflect what the responsibilities are? e.g. IQueueClient.

Reply:
Yep.